### PR TITLE
Update web site for 1.6.0 release and add 1.6.0 release announcement

### PR DESCRIPTION
**Pull request by tillrohrmann**
This PR updates the Flink web site for the Flink 1.6.0 release and adds the Flink 1.6.0 release announcement.

### CODE REVIEW

File: _posts/2018-08-08-release-1.6.0.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+## New Features and Improvements
+
+### Improving Flink's State Support
+
+* **Support for State TTL** ([FLINK-9510](https://issues.apache.org/jira/browse/FLINK-9510)):
+This feature allows to specify a time-to-live (TTL) for Flink state. 
+Once the time-to-live has been exceeded Flink will no longer give access to the respective state values. 
+The expired data is cleaned up on access so that the operator keyed state doesn’t grow infinitely.
+This feature fully complies with new data protection regulations (e.g. GDPR).
```
Comment (aljoscha) [inline _posts/2018-08-08-release-1.6.0.md]: Also mention here the deletion-on-scan, I think.

File: _posts/2018-08-08-release-1.6.0.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+* **Faster Timer Deletions** ([FLINK-9423](https://issues.apache.org/jira/browse/FLINK-9423)): 
+Improving Flink’s internal timer data structure such that the deletion complexity is reduced from O(n) to O(log n). 
+This significantly improves Flink jobs using Flink’s timers. 
+Deleting timers are also exposed through a user-facing API now.
+
+### Extending Flink's Deployment Options
+
+* **Job Cluster Container Entrypoint** ([FLINK-9488](https://issues.apache.org/jira/browse/FLINK-9488)):
+Flink 1.6.0 provides an easy-to-use container entrypoint to bootstrap a job cluster. 
+Combining this entrypoint with a user code jar creates a self-contained image which automatically executes the contained Flink job when deployed.
```
Comment (aljoscha) [inline _posts/2018-08-08-release-1.6.0.md]: Nitpick, but I think it should be "user-code jar"

File: _posts/2018-08-08-release-1.6.0.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+### Extending Flink's Deployment Options
+
+* **Job Cluster Container Entrypoint** ([FLINK-9488](https://issues.apache.org/jira/browse/FLINK-9488)):
+Flink 1.6.0 provides an easy-to-use container entrypoint to bootstrap a job cluster. 
+Combining this entrypoint with a user code jar creates a self-contained image which automatically executes the contained Flink job when deployed. 
+Since the image already contains the Flink job, client communication is no longer necessary. 
+Avoiding additional communication steps with the client reduces the number of moving parts and, improves operations in a container environment significantly.
+
+* **Fully RESTified Job Submission** ([FLINK-9280](https://issues.apache.org/jira/browse/FLINK-9280)):
+The Flink client now sends all job relevant content via a single POST call to the server.
```
Comment (aljoscha) [inline _posts/2018-08-08-release-1.6.0.md]: Same here, I think it should be "job-relevant content"

File: _posts/2018-08-08-release-1.6.0.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+* **User-Defined Function in SQL Client CLI** ([FLINK-8863](https://issues.apache.org/jira/browse/FLINK-8863)):
+The SQL Client CLI now supports the registration of user-defined functions. 
+This considerably improves the CLI’s expressiveness, because SQL queries can be enriched with more powerful custom table, aggregate, and scalar functions.
+
+* **Support for Batch Queries in SQL Client CLI** ([FLINK-8861](https://issues.apache.org/jira/browse/FLINK-8861)):
+The SQL Client CLI now supports the execution of batch queries.
+
+* **Support for INSERT INTO Statements in SQL Client CLI** ([FLINK-8858](https://issues.apache.org/jira/browse/FLINK-8858)):
+By supporting SQL’s INSERT INTO statements, the SQL Client CLI can be used to submit long-running SQL queries to Flink that sink their results in external systems. 
+The SQL Client itself can be shutdown after submission.
```
Comment (aljoscha) [inline _posts/2018-08-08-release-1.6.0.md]: I think it should be "can be shut down"

File: _posts/2018-08-08-release-1.6.0.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title:  "Apache Flink 1.6.0 Release Announcement"
+date:   2018-08-08 18:30:00
```
Comment (zentol) [inline _posts/2018-08-08-release-1.6.0.md]: AFAIK we wanted to keep use a consistent time (12:00:00), but I don't really know why. Just saw this comment in another PR.

File: _posts/2018-08-08-release-1.6.0.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+* **Support for Batch Queries in SQL Client CLI** ([FLINK-8861](https://issues.apache.org/jira/browse/FLINK-8861)):
+The SQL Client CLI now supports the execution of batch queries.
+
+* **Support for INSERT INTO Statements in SQL Client CLI** ([FLINK-8858](https://issues.apache.org/jira/browse/FLINK-8858)):
+By supporting SQL’s INSERT INTO statements, the SQL Client CLI can be used to submit long-running SQL queries to Flink that sink their results in external systems. 
+The SQL Client itself can be shutdown after submission. 
+Table sinks can be defined in a YAML file using the new unified table sink properties.
+
+* **Unified Table Sinks and Formats** ([FLINK-8866](https://issues.apache.org/jira/browse/FLINK-8866), [FLINK-8558](https://issues.apache.org/jira/browse/FLINK-8558)):
+In the past, table sinks had to be configured programmatically.
```
[the remark this answers is not in the document]
Comment (tillrohrmann) [reply _posts/2018-08-08-release-1.6.0.md]: I'll reword it a bit.

File: _posts/2018-08-08-release-1.6.0.md [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+Both SQL and Table API now include more built-in functions such as `MD5, SHA1, SHA2, LOG`, and `UNNEST` for multisets.
+
+### More Connectors
+
+* **New StreamingFileSink** ([FLINK-9750](https://issues.apache.org/jira/browse/FLINK-9750)):
+The new `StreamingFileSink` is an exactly-once sink for writing to filesystems which capitalizes on the knowledge acquired from the previous `BucketingSink`. 
+Exactly-once is supported through integration of the sink with Flink’s checkpointing mechanism.
+The new sink is built upon Flink’s own `FileSystem` abstraction and it supports local file system and HDFS, with plans for S3 support in the near future.
+It exposes pluggable file rolling and bucketing policies.
+Apart from row-wise encoding formats, it exposes APIs for bulk-encoding formats like Parquet/ORC and it already has support for Parquet.
```
[the remark this answers is not in the document]
Comment (tillrohrmann) [reply _posts/2018-08-08-release-1.6.0.md]: True.

File: _posts/2018-08-08-release-1.6.0.md (hunk 5 above)
Comment (tillrohrmann) [reply _posts/2018-08-08-release-1.6.0.md]: Alright, will update the blog post accordingly.

**Comment by tillrohrmann** [comment]
Thanks for the review @aljoscha and @zentol. I've addressed your comments and split the original commit into a new commit and the rebuild commit.

File: _posts/2018-08-08-release-1.6.0.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+## New Features and Improvements
+
+### Improving Flink's State Support
+
+* **Support for State TTL** ([FLINK-9510](https://issues.apache.org/jira/browse/FLINK-9510), [FLINK-9938](https://issues.apache.org/jira/browse/FLINK-9938)):
+This feature allows to specify a time-to-live (TTL) for Flink state. 
+Once the time-to-live has been exceeded Flink will no longer give access to the respective state values. 
+The expired data is cleaned up on access so that the operator keyed state doesn’t grow infinitely and it won't be included in subsequent checkpoints.
+This feature fully complies with new data protection regulations (e.g. GDPR).
+
```
Comment (twalthr) [inline _posts/2018-08-08-release-1.6.0.md]: Remove empty line?

**Pull request merged**